lan867x/idf_component.yml

```diff
@@ -0,0 +1,7 @@
+version: "0.9.0"
+targets:
+  - esp32
+description: LAN867x Ethernet PHY Driver
+url: https://github.com/espressif/esp-eth-drivers/tree/master/lan867x
+dependencies:
+  idf: ">=5.0"
```

Comment:
See https://github.com/espressif/esp-eth-drivers/pull/19/files#r1484098535

Reply:
Changed to `idf: ">=4.4"` because it will eventually be backported.